x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.ts

```diff
@@ -368,7 +368,11 @@ export function registerIndexRoutes({
         if ((error as Error).message === ErrorCode.PIPELINE_ALREADY_EXISTS) {
           return createError({
             errorCode: (error as Error).message as ErrorCode,
-            message: 'Pipeline already exists',
+            message: `
+              A pipeline with the name ${getPrefixedInferencePipelineProcessorName(pipelineName)}
```

Comment:
```suggestion
              A pipeline with the name "${getPrefixedInferencePipelineProcessorName(pipelineName)}"
```

Reply:
I think the copy is fantastic. The only thing I'd suggest is wrapping the pipeline name in quotes. Its long and bleeds into the rest of the sentence a wee bit.